docs/resources/aws_flow_log.md.erb

```diff
@@ -0,0 +1,82 @@
+---
+title: About the aws_flow_log Resource
+platform: aws
+---
+
+# aws\_flow\_low
+
+Use the `aws_flow_log` InSpec audit resource to test properties of a single Flow Log.
+
+## Syntax
+
+    describe aws_flow_log('fl-9c718cf5') do
+      it { should exist }
+    end
+
+## Resource Parameteres
+### flow_log_id
+
+This resource expects a single parameter that uniquely identifies the Flow Log. You may pass it as a string, or as the value in a hash:
+
+    describe aws_flow_log('fl-9c718cf5') do
+      it { should exist }
+    end
+
+    describe aws_flow_log(flow_log_id: 'fl-8905f8e0') do
+      it { should exist }
+    end
+
+### subnet_id
+
+To search for a flow log by the associated subnet id:
+
+    describe aws_flow_log(subnet_id: 'subnet-c6a4319c') do
+      it { should exist }
+    end
+
+### vpc_id
+
+To search for a flow log by the associated vpc id:
+
+    describe aws_flow_log(vpc_id: 'vpc-96cabaef') do
+      it { should exist }
+    end
+
+## Matchers
+
+For a full list of available matchers, please visit our [matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+### name
```

Comment:
I think we should refer to this as `flow_log_id`, not as name.  I didn't see anything about a name attribute in the describe_flow_logs API docs.  Don't want to introduce confusion :)